content/en-us/reference/engine/classes/Debris.yaml

```diff
@@ -3,64 +3,73 @@ type: class
 category: Utility
 memory_category: Instances
 summary: |
-  The Debris service allows the developer to schedule the removal of the object
-  without yielding any code, through the usage of the `Class.Debris:AddItem()`
+  The Debris service allows the developer to schedule guaranteed destruction of
+  the object without yielding, through the usage of the `Class.Debris:AddItem()`
   method.
 description: |
-  The Debris service allows the developer to schedule the removal of the object
-  without yielding any code, through the usage of the `Class.Debris:AddItem()`
+  The Debris service allows the developer to schedule guaranteed destruction of
+  the object without yielding, through the usage of the `Class.Debris:AddItem()`
   method.
 
-  After the lifetime argument has elapsed (in seconds) the object is removed in
-  the same manner as `Class.Instance:Destroy()`.
+  After the lifetime argument has elapsed (in seconds) the object is destroyed
+  in the same manner as `Class.Instance:Destroy()`.
 
   As Debris is a service it must be created using the
   `Class.ServiceProvider:GetService()` method.
```

Comment:
```suggestion
```

Reply:
Can we omit this? I don't recall seeing other services making this specific usage note.

Reply:
Just left it in from before, but yes I agree it doesn't need to be here if it's not on other service pages.